table/server/master/src/main/java/alluxio/master/table/Database.java

```diff
@@ -484,7 +566,8 @@ public void remove() {
     Journal.JournalEntry entry = Journal.JournalEntry.newBuilder().setUpdateDatabaseInfo(
         toJournalProto(getDatabaseInfo(), mName)).build();
     return CloseableIterator.noopCloseable(
-        Iterators.concat(Iterators.singletonIterator(entry), getTableIterator()));
+        Iterators.concat(Iterators.singletonIterator(entry),
+            getTableIterator(), getTablePartitionsIterator()));
```

Comment:
Is it perfectly fine output all the tables and then all the individual partition batches?

Reply:
The table iterator change to include partition iterators so they are now placed together. This way seems to be safer